### PR TITLE
Registration page Improvments and format

### PR DESCRIPTION
**Pull request by Dmitriy-Kovalenko1**
Change from open Registration page URL to Home Page - click registration link - Go to Registration Page. Dmitriy & Irina

### CODE REVIEW

File: modules/user/registration/registration_positive.spec.js [hunk 1]
```diff
@@ -4,105 +4,108 @@ import {newUser, pageRegister} from './registration_data';
 import LoginPage from '../../_PageObjects/LoginPage';
 import ProfilePage from '../../_PageObjects/ProfilePage';
 import Logout from '../../_PageObjects/Logout';
+import HomePage from "../../_PageObjects/HomePage";
```
Comment (fiore24) [inline modules/user/registration/registration_positive.spec.js]: single quotes

File: modules/user/registration/registration_negativeUserAlreadyExist.spec.js [hunk 2]
```diff
@@ -2,73 +2,76 @@ import {expect} from 'chai';
 import RegistrationPage from '../../_PageObjects/RegistrationPage';
 import {newUser, pageRegister} from './registration_data';
 import {userRegistered} from './userAlreadyRegister';
+import HomePage from "../../_PageObjects/HomePage";
```
Comment (fiore24) [inline modules/user/registration/registration_negativeUserAlreadyExist.spec.js]: single quotes

File: modules/user/registration/registration_negativeEmptyPhone.spec.js [hunk 3]
```diff
@@ -1,63 +1,66 @@
 import {expect} from 'chai';
 import RegistrationPage from '../../_PageObjects/RegistrationPage';
 import {newUser, pageRegister} from './registration_data';
+import HomePage from "../../_PageObjects/HomePage";
```
Comment (fiore24) [inline modules/user/registration/registration_negativeEmptyPhone.spec.js]: single quotes

File: modules/user/registration/registration_negativeEmptyPassword.spec.js [hunk 4]
```diff
@@ -1,63 +1,66 @@
 import {expect} from 'chai';
 import RegistrationPage from '../../_PageObjects/RegistrationPage';
 import {newUser, pageRegister} from './registration_data';
+import HomePage from "../../_PageObjects/HomePage";
```
Comment (fiore24) [inline modules/user/registration/registration_negativeEmptyPassword.spec.js]: single quotes

File: modules/user/registration/registration_negativeEmptyGoals.spec.js [hunk 5]
```diff
@@ -1,63 +1,66 @@
 import {expect} from 'chai';
 import RegistrationPage from '../../_PageObjects/RegistrationPage';
 import {newUser, pageRegister} from './registration_data';
+import HomePage from "../../_PageObjects/HomePage";
```
Comment (fiore24) [inline modules/user/registration/registration_negativeEmptyGoals.spec.js]: single quotes

File: modules/user/registration/registration_negativeEmptyLastName.spec.js [hunk 6]
```diff
@@ -1,63 +1,66 @@
 import {expect} from 'chai';
 import RegistrationPage from '../../_PageObjects/RegistrationPage';
 import {newUser, pageRegister} from './registration_data';
+import HomePage from "../../_PageObjects/HomePage";
```
Comment (fiore24) [inline modules/user/registration/registration_negativeEmptyLastName.spec.js]: single quotes

**Comment by fiore24** [comment]
single quotes for import HomePage from "../../_PageObjects/HomePage"; in all tests

File: modules/user/registration/registration_positive.spec.js (hunk 1 above)
Comment (Dmitriy-Kovalenko1) [reply modules/user/registration/registration_positive.spec.js]: All DONE!

File: modules/user/registration/registration_negativeUserAlreadyExist.spec.js (hunk 2 above)
Comment (Dmitriy-Kovalenko1) [reply modules/user/registration/registration_negativeUserAlreadyExist.spec.js]: Done.

File: modules/user/registration/registration_negativeEmptyPhone.spec.js (hunk 3 above)
Comment (Dmitriy-Kovalenko1) [reply modules/user/registration/registration_negativeEmptyPhone.spec.js]: Done.

**Pull request merged**